package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "json-schema-to-ts",
-  "version": "2.6.2-beta.0",
+  "version": "2.6.2-beta.1",
```

Comment:
You don't need to do it in the PR, the version is updated in my release script. Can you remove this change ?

Reply:
Done.